CODEOWNERS

```diff
@@ -0,0 +1,12 @@
+# Lines starting with '#' are comments.
```

Comment:
I'm not sure if these comments are necessary..
Could you please remove these or replace these with new comments to fit our situation.